SFUCommute/sfu_commute_V.2/SignupPage.swift

```diff
@@ -232,7 +232,7 @@ class SignupPage: UIViewController {
                 self.tips.show(forView: self.emailTextField)
                 
             }
-        }
+        }.resume()
```

Comment:
Swift is a blocking language unlike Javascript... if we do this call without Alamofire, then we need to make it async, like what you are doing here.. But Alamofire wraps the response handler in a "closure", which is a callback function. So the request actually does not block the code, the response callback function is called whenever we recieve the response.

Good thinking but I think `resume()` is not needed. Correct me if I'm wrong... I've been writing too much non-blocking Node.js code, and forgot the async problem for other languages! :)